Sprint-1/interpret/time-format.js

```diff
@@ -13,12 +13,24 @@ console.log(result);
 
 // a) How many variable declarations are there in this program?
 
+// There are about six variable declarations 
+
 // b) How many function calls are there?
 
+// There is a only one function call..the console.log function
+
 // c) Using documentation, explain what the expression movieLength % 60 represents
 
+// This expression represents a calculation that will return a remainder of movieLength seconds after it's divided by 60. The (%) modulus operator is used to find the remainder after dividing one number by another.  
```

Comment:
This is a literal translation of the code; it does not give any insight what the code is trying to calculate. Can you describe what the code is trying to calculate, in lament terms?